dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -214,13 +214,31 @@ public List<T> findMyEntries(long userId) {
     public List<T> findMyEntriesPublished(long userId) {
         return (List<T>) this.currentSession().createNamedQuery("io.dockstore.webservice.core." + typeOfT.getSimpleName() + ".getPublishedEntriesByUserId", Entry.class).setParameter("userId", userId).list();
     }
+    public List<CollectionEntry> getAllCollectionWorkflows(long collectionId) {
```

Comment:
I suggest either a JavaDoc clarifying this is returning workflows in the JPA sense (because it returns App Tools as well, which is not clear from the method name). Or renaming the method and possibly the JPA query.

I can't think of a good name though, and I'm even having a hard time formulating my comment. :)

Reply:
javadoc added